lib/sacastats_web/controllers/character_controller.ex

```diff
@@ -219,4 +239,28 @@ defmodule SacaStatsWeb.CharacterController do
 
   def get_kills_to_next_medal(total_kills),
     do: 10 - total_kills
+
+  def get_character_sex(faction_id, _head_id) when faction_id == 0 or faction_id == 4,
```

Comment:
Not a big thing, but if you want to shorten the guard a bit, you can do
```suggestion
  def get_character_sex(faction_id, _head_id) when faction_id in [0, 4]
```